public/js/app/counter.js

```diff
@@ -0,0 +1,26 @@
+const delay = 3000;
```

Comment:
But in fact, I **really** don't like to ask database twice every 3 seconds. Make it once per 30 seconds.
Don't forget, that database is our big bottleneck, we prefer to not to make any unnecessary load on it.

Reply:
(as discussed in zulip this is N/A )